modin/pandas/test/dataframe/test_default.py

```diff
@@ -132,7 +132,12 @@ def test_to_numpy(data):
 @pytest.mark.parametrize("data", test_data_values, ids=test_data_keys)
 def test_partition_to_numpy(data):
     frame = pd.DataFrame(data)
-    for partition in frame._query_compiler._modin_frame._partitions.flatten().tolist():
+    qc = frame._query_compiler
+    if get_current_execution() == "Client":
```

Comment:
Similar code is duplicated multiple times. What about having the _modin_frame property in ClientQueryCompiler?
Something like:

    @property
    def _modin_frame(self):
        return self._service._qc[self._id]._modin_frame

Reply:
I think this code only appears twice. Also, the client query compiler will not necessarily be backed by a query compiler that has a `_modin_frame`. I prefer leaving the code as is.